UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Utils/AtmosUtils.cs

```diff
@@ -24,6 +24,37 @@ public static GasValues GetGasValues()
 			return new GasValues();
 		}
 
+
+		public static List<List<GasValues>> PooledGasValuesLists = new List<List<GasValues>>();
```

Comment:
Codacy is saying make this private

Reply:
PooledGasValuesLists.Add(List); 
			problem it's accessed everywhere so I don't know what Codacy  is on

Reply:
Could make it readonly.